### DISCUSSION TITLE
Multiple admob ad units for waterfall

### DISCUSSION THREAD

**Discussion by yvant** [distriqt/ANE-IronSource#58] [Q&A]
Hi, 
I just had a call from IronSource to discuss about mediation. They told me I needed to add multiple ad placements in IronSource (around 8) from AdMob, so I created them and added them to the IronSource backend. Do I need to do something else regarding the application? Do I need to publish it again or should it be already taken into account?

**Comment by marchbold** [comment]
That should all be handled by ironSource backend as far as I'm aware, unless I've misunderstood what you are doing.
